### PR TITLE
[MI-2177][Documentation]: Updated readme files

### PR DESCRIPTION
**Pull request by avas27JTG**
Updated readme files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This plugin integrates the services of Azure DevOps in Mattermost. For a stable
 This plugin contains the following features:
 - Right-hand sidebar (RHS) shows the list of linked projects and subscriptions created for each project with separate buttons to connect an account, link or unlink a project and add or delete a subscription.
 
-- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user is connected to his Azure DevOps account.
+- Preview of Work item or Pull request URL: A preview of the work item or pull request for a linked project will be created when their respective URL is posted in a channel and the user is connected to his Azure DevOps account.
```
Comment (ayusht2810) [inline README.md:24]:
```suggestion
- Preview of the Work item or Pull request URL: A preview of the work item or pull request for a linked project will be created when their respective URLs are posted in a channel and the user is connected to his Azure DevOps account.
```

File: README.md [hunk 2]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
 - Create work items: A work item can be created using the slash command below.
 
     ```
-    - /azuredevops boards create
+    - /azuredevops boards create [title] [description]
     ```
     On successful creation of a work item, you will get a message from the bot with the details of the newly created work item.
 
-- Add subscriptions: A user can create subscriptions for a linked project to get messages in a selected channel for selected events on work items like Create, Update and Delete.
+- Add subscriptions: A user can create subscriptions for a linked project to get messages in a selected channel for selected events on work items and pull requests.
```
Comment (ayusht2810) [inline README.md:48]:
```suggestion
- Add subscriptions: A user can create subscriptions for a linked project to get notifications in a selected channel for selected events on work items and pull requests.
```

Comment (ayusht2810) [reply README.md:48]: maybe?

File: docs/developer_docs.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ This plugin integrates the services of Azure DevOps in Mattermost. For a stable
 This plugin contains the following features:
 - Right-hand sidebar (RHS) shows the list of linked projects and subscriptions created for each project with separate buttons to connect an account, link or unlink a project and add or delete a subscription.
 
-- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user is connected to his Azure DevOps account.
+- Preview of Work item or Pull request URL: A preview of the work item or pull request for a linked project will be created when their respective URL is posted in a channel and the user is connected to his Azure DevOps account.
```
Comment (ayusht2810) [inline docs/developer_docs.md:25]:
```suggestion
- Preview of the Work item or Pull request URL: A preview of the work item or pull request for a linked project will be created when their respective URLs are posted in a channel and the user is connected to his Azure DevOps account.
```

File: docs/developer_docs.md [hunk 4]
```diff
@@ -42,29 +42,62 @@ This plugin contains the following features:
 - Create work items: A work item can be created using the slash command below.
 
     ```
-    - /azuredevops boards create
+    - /azuredevops boards create [title] [description]
     ```
     On successful creation of a work item, you will get a message from the bot with the details of the newly created work item.
 
-- Add subscriptions: A user can create subscriptions for a linked project to get messages in a selected channel for selected events on work items like Create, Update and Delete.
+- Add subscriptions: A user can create subscriptions for a linked project to get messages in a selected channel for selected events on work items and pull requests.
```
Comment (ayusht2810) [inline docs/developer_docs.md:49]:
```suggestion
- Add subscriptions: A user can create subscriptions for a linked project to get notifications in a selected channel for selected events on work items and pull requests.
```

**Comment by avas27JTG** [comment]
@ayusht2810 no need to review both the developer doc and main readme files as the same changes of the main readme file is copied in the developer doc

File: README.md [hunk 5]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
+
+    ```
+    - /azuredevops repos subscription list anyone all_channels 
+    ```
+
+    Supported filters on the above slash command:
+    - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
+    - Show for all channels: when the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed, you can leave this to list the subscriptions of the current channel only.
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
```
Comment (raghavaggarwal2308) [inline README.md:82]:
```suggestion
    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
```
I think our motive here is `only admins can view/list subscriptions` but its sounding like `admins can only view/list subscription`

File: README.md [hunk 6]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
+    - /azuredevops boards subscription delete [subscription id]
     ```
 
+    - For deleting Repos subscriptions
+
+    ```
+    - /azuredevops repos subscription delete [subscription id]
+    ```
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only create/delete a subscription.
```
Comment (raghavaggarwal2308) [inline README.md:98]: same

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (avas27JTG) [reply README.md:82]: yes but we need to read the entire sentence to get the complete meaning `admins can only view/list subscription in a channel where he is not a member`

File: README.md (hunk 6 above)
Comment (avas27JTG) [reply README.md:98]: yes this also holds true only Admins can create/delete subscription

File: README.md (hunk 5 above)
Comment (avas27JTG) [reply README.md:82]: both the interpretation are same here

**Review by raghavaggarwal2308 — APPROVED** [review]

**Review by ayusht2810 — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -21,7 +21,7 @@ This plugin integrates the services of Azure DevOps in Mattermost. For a stable
 This plugin contains the following features:
 - Right-hand sidebar (RHS) shows the list of linked projects and subscriptions created for each project with separate buttons to connect an account, link or unlink a project and add or delete a subscription.
 
-- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user is connected to his Azure DevOps account.
+- Preview of the Work item or Pull request URL: A preview of the work item or pull request for a linked project will be created when their respective URLs are posted in a channel and the user is connected to his/her Azure DevOps account.
```
Comment (manojmalik20) [inline README.md:24]:
```suggestion
- Preview of the work item or pull request: A preview of the work item or pull request for a linked project will be created when their respective URLs are posted in a channel and the user is connected to his/her Azure DevOps account.
```

File: README.md [hunk 8]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
-- View/List subscriptions: A user can view the list of subscriptions for a project by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
+    - For creating Repos subscriptions
 
     ```
-    - /azuredevops boards subscriptions
+    - /azuredevops repos subscription add
     ```
 
-- Delete subscriptions: A user can delete subscriptions for a project from RHS by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also delete a subscription for a project by using the slash command below.
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only create/delete a subscription.
```
Comment (manojmalik20) [inline README.md:62]:
```suggestion
    **Note:** Only Mattermost users who are project admins or team admins on the linked Azure DevOps project can create/delete a subscription.
```

File: README.md [hunk 9]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
+
+- View/List subscriptions: A user can view the list of subscriptions for a project by going to the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
```
Comment (manojmalik20) [inline README.md:64]:
```suggestion
- View/List subscriptions: A user can view the list of subscriptions for a project by going to the subscriptions list page after clicking on the project title under "Linked Projects" in the right-hand sidebar. Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
```

File: README.md [hunk 10]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
+
+    - For listing Repos subscriptions
+
+    ```
+    - /azuredevops repos subscription list anyone all_channels 
+    ```
+
+    Supported filters on the above slash command:
+    - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
+    - Show for all channels: when the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed, you can leave this to list the subscriptions of the current channel only.
```
Comment (manojmalik20) [inline README.md:80]:
```suggestion
    - Show for all channels: When the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed. You can leave this empty to list the subscriptions of the current channel only.
```

File: README.md (hunk 5 above)
Comment (manojmalik20) [inline README.md:82]:
```suggestion
    **Note:** Only Mattermost users who are project admins or team admins on the linked Azure DevOps project can view/list subscriptions that exist in a channel where even they are not a member.
```

File: README.md [hunk 11]
```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
...
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
+
+- Delete subscriptions: A user can delete subscriptions for a project from RHS by going to the subscriptions list page after clicking on the project title under "Linked Projects". Users can also delete a subscription for a project by using the slash command below.
+
+    - For deleting Boards subscriptions
 
     ```
-    - /azuredevops boards unsubscribe [subscription id]
+    - /azuredevops boards subscription delete [subscription id]
```
Comment (manojmalik20) [inline README.md:89]:
```suggestion
    /azuredevops boards subscription delete [subscription id]
```
Why have you added hyphens everywhere before all commands?

File: README.md (hunk 6 above)
Comment (manojmalik20) [inline README.md:98]:
```suggestion
    **Note:** Only Mattermost users who are project admins or team admins on the linked Azure DevOps project can create/delete a subscription.
```

File: README.md [hunk 12]
```diff
@@ -50,52 +50,52 @@ To add a new subscription for a linked project click on the project title under
...
 
     ```
-    - /azuredevops repos subscription list anyone all_channels 
+    /azuredevops repos subscription list anyone all_channels 
     ```
 
     Supported filters on the above slash command:
     - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
-    - Show for all channels: when the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed, you can leave this to list the subscriptions of the current channel only.
+    - Show for all channels: When the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed. You can skit this filter param to list the subscriptions of the current channel only.
```
Comment (manojmalik20) [inline README.md:80]:
```suggestion
    - Show for all channels: When the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed. You can skip this filter param to list the subscriptions of the current channel only.
```

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**